helpers/helper-translation-discussion.php

```diff
@@ -52,6 +52,12 @@ public function register_post_type_and_taxonomy() {
 			'single'       => true,
 			'show_in_rest' => true,
 		) );
+
+		register_meta( 'comment', 'comment_topic', array(
+			'description'  => 'Reason for the comment',
+			'single'       => true,
+			'show_in_rest' => true,
+		) );
```

Comment:
We need to add a `sanitize_callback` here and also to the other `register_meta` calls, see http://developer.wordpress.org/reference/functions/register_meta/

Reply:
Added `sanitize_callback` and TODO comments for the other `register_meta` calls